pkg/client/typed/dynamic/client.go

```diff
@@ -83,6 +83,13 @@ func (c *Client) Resource(resource *unversioned.APIResource, namespace string) *
 	}
 }
 
+func (c *Client) Subresource(resource *unversioned.APIResource, subresource string, namespace string) *SubresourceClient {
```

Comment:
I have a mild preference for what @deads2k describes. I also don't mind having both ways.